src/sys/signalfd.rs

```diff
@@ -60,10 +60,10 @@ pub fn signalfd(fd: RawFd, mask: &SigSet, flags: SfdFlags) -> Result<RawFd> {
 /// # Examples
 ///
 /// ```
+/// use nix::sys::signal::*;
```

Comment:
Let's make these `use` lines not appear in the docs by prepending with `#` since we're modifying this example.